speech/google/cloud/speech/operation.py

```diff
@@ -14,120 +14,55 @@
 
 """Long running operation representation for Google Speech API"""
 
-from google.cloud.speech.metadata import Metadata
-from google.cloud.speech.transcript import Transcript
+from google.cloud.grpc.speech.v1beta1 import cloud_speech_pb2
```

Comment:
@tseaver @daspecster This effectively makes the `_HAVE_GAX` / `ImportError` check in `client` pointless. And it is anyway since we have the deps in `setup.py`. I am :+1: on removing the check and only using the env. var. for `_USE_GAX`.